.bazelrc

```diff
@@ -19,3 +19,7 @@ build \
 build:ios --define=manual_stamp=manual_stamp
 
 build:android --fat_apk_cpu=x86,x86_64,armeabi-v7a,arm64-v8a
+
+build:release \
+  --compilation_mode=opt \
```

Comment:
This is present in the OSS .bazelrc:

```
# Optimize build for binary size reduction.
build:sizeopt -c opt --copt -Os
```

I think this is what we want for now. I'm not sure if the OSS .bazelrc can be imported into this one somehow and we can just use that? @keith?

Reply:
Also while we are here I would also add `--copt --gdb3` so we have full symbols across all deps for ease of debugging.

Reply:
Yopu can import the config from envoy, https://docs.bazel.build/versions/master/guide.html#imports and then you could do something like `build:release --config=whateverfromenvoy`

Reply:
Nice, done!

Reply:
You can remove the above line now, since it's implied by the next line.

Reply:
Updated